lib/compliance/parsepdf417-ca-bc.js

```diff
@@ -0,0 +1,94 @@
+var _ = require('lodash/fp')
+
+module.exports.parse = function parse (data) {
+  const track1Start = data.indexOf('%')
+  const track2Start = data.indexOf(';')
+  const track3Start = data.indexOf('_%')
+
+  if (track1Start === -1 || !track2Start === -1 || !track3Start === -1) return null
+
+  return parseRaw(data)
+}
+
+function parseRaw (data) {
+  const track1Start = data.indexOf('%')
```

Comment:
We shouldn't do these searches twice. If we merge parse and parseRaw, the combined function is just one line longer than parseRaw (the ``if`` statement).